### ISSUE TITLE
Arrow keys should nudge position

### ISSUE THREAD

**Issue by nheagy** [spritebuilder/SpriteBuilder#1316]
The arrow keys should nudge the position of the currently selected node in 1pt increments.

Ideally, this would allow support Shift-Nudge to jump by 10 points.

**Comment by NickyWeber** [comment]
+1

**Comment by efusco** [comment]
+1 to this, I would love to have this feature!!

**Comment by stuartcarnie** [comment]
This is already supported under the Object Nudge / Move menu items. Nudge is ⌘+arrow for 1pt and Move is ⌘+⇧+arrow for 10pt

**Comment by NickyWeber** [comment]
Exactly and it's pretty annoying that you have to hold a modifier key while arrow keys alone just don't do anything.

**Comment by cocojoe** [comment]
Arrows keys alone will navigate the node hierarchy in timeline.

**Comment by NickyWeber** [comment]
@cocojoe thanks. still as a default behavior I'd rather move nodes on stage.

**Comment by stuartcarnie** [comment]
Easy enough to fix – want me to take it?

**Comment by NickyWeber** [comment]
It's a product decision we cannot make, I'll ask Vik next week.

**Comment by vlidholt** [comment]
We have looked into this one before, and is not at easy as it sounds. The arrow keys are used for other purposes such as moving between selections in the list and table views and when editing properties in the text boxes. The Cmd-arrow keys are used by other applications such as Photoshop, so most users are already used to it.

**Comment by stuartcarnie** [comment]
I agree @vlidholt – the feature is already there and useful as is, even if one has to use the modifier keys. 

I suspect the complexity is because the events are handled globally (on the application delegate), rather than using the responder chain. If the Cocos2D view becomes the first responder, we can handle those locally.

I would like to see the arrow keys work intuitively based on the current focus (which is consistent with other applications)
- if I'm editing a numeric field, then up / down should increment / decrement the value
- if I've selected the scrubber, it should move forwards / backwards in time
- If I've selected a node in the level editor, it should nudge the item

**Comment by nheagy** [comment]
Actually @vlidholt Photoshop uses Arrows without Cmd, it happens to work because Cmd-Arrow is unassigned.

It is because of the supremacy of the arrow keys that most video/animation apps use , and . to move the scrubber, @stuartcarnie. This way you can both move objects and the scrubber without worrying about focus.

**Comment by stuartcarnie** [comment]
@nheagy I like the `,` and `.` shortcuts
